low-level/files/opt/seagate/sspl/setup/sspl_reset.py

```diff
@@ -2,89 +2,56 @@
 import shutil
 
 from framework.base.sspl_constants import (PRODUCT_FAMILY,
-                                           file_store_config_path,
-                                           SSPL_CONFIGURED,
                                            DATA_PATH,
                                            SSPL_CONFIG_INDEX,
                                            sspl_config_path)
-from cortx.utils.process import SimpleProcess
 from cortx.utils.service import DbusServiceHandler
 from cortx.utils.conf_store import Conf
-from files.opt.seagate.sspl.setup.setup_error import SetupError
 from files.opt.seagate.sspl.setup.setup_logger import logger
 
+
 class HardReset:
     """Performs SSPL Hard reset.
-    'hard' is used to reset configs and clean log directory """
+    This Interface is used to reset configs and clean log files """
 
     name = "hard"
-    user_present=False
+    user_present = False
 
     def del_file(self, filename):
         if os.path.exists(filename):
             os.remove(filename)
 
+    @classmethod
+    def del_files_from_dir(cls, fformat, dir_path):
+        for root, _, files in os.walk(dir_path):
+            for file in files:
+                if file.endswith(fformat):
+                    os.remove(os.path.join(root, file))
+
     def process(self):
-        # stop sspl service
         dbus_service = DbusServiceHandler()
-        dbus_service.stop('sspl-ll.service')
-
-        # Remove sspl_conf
-        self.del_file(file_store_config_path)
-
-        # Remove sspl-configured file
-        self.del_file(SSPL_CONFIGURED)
+        # Stop SSPL service if state is active
+        service_state = dbus_service.get_state('sspl-ll.service')
+        if service_state._state == 'active':
+            logger.info(f'For {self.name}, SSPL service should be stopped'
```

Comment:
logger.warning ("SSPL service should have been stopped before Factory reset is invoked, stopping now")

Reply:
Done